_includes/commands/advtable-cmds.md

```diff
@@ -5,8 +5,9 @@
 | mceSortTableAdvanced     | Performs an Advanced Table Sort. See below for details.                                         |
 | mceSortTableByColumnAsc  | Sorts the current table ascending by column based on the current cursor position or selection.  |
 | mceSortTableByColumnDesc | Sorts the current table descending by column based on the current cursor position or selection. |
-| mceSortTableByRowAsc    | Sorts the current table ascending by row based on the current cursor position or selection.     |
-| mceSortTableByRowDesc     | Sorts the current table descending by row based on the current cursor position or selection.    |
+| mceSortTableByRowAsc     | Sorts the current table ascending by row based on the current cursor position or selection.     |
+| mceSortTableByRowDesc    | Sorts the current table descending by row based on the current cursor position or selection.    |
+| mceTableToggleSeries     | Toggles a series column on the selected table. See below for more details. |
```

Comment:
This will probably need the `{{ site.requires_5_7v }}` part here as well, to show it's only available from 5.7 onwards.

Reply:
> See below for more details.

Change to form of "For details, see []()." 

It wont always be below....